libr/core/cmd_anal.c

```diff
@@ -1825,6 +1826,13 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 			if (fcn) {
 				RAnalRef *ref;
 				RListIter *iter;
+				SdbList *secs = r_io_section_vget_secs_at (core->io, fcn->addr);		//use map-API here
```

Comment:
this operation is done several times, what about having r_io-section_vget_secs_at_peek()... with obviously a better and shorter name